Makefile

```diff
@@ -7,7 +7,7 @@ IMAGE ?= $(IMAGE_TAG_BASE):$(VERSION)
 DEPLOYDIR = ./deploy
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.23
+ENVTEST_K8S_VERSION = latest
```

Comment:
With this configuration, this version is resolved:

![Screenshot 2025-04-21 at 11 07 18 PM](https://github.com/user-attachments/assets/57146f82-594e-436a-af5e-d97169e234b0)